src/RA_MemValue.cpp

```diff
@@ -25,12 +26,12 @@ double MemValue::Clause::GetValue() const
 
     if (m_nSecondAddress != 0)
     {
-        unsigned int nSecondVal = g_MemManager.ActiveBankRAMRead(m_nSecondAddress, m_nSecondVarSize);
+        auto nSecondVal = static_cast<std::uint64_t>(g_MemManager.ActiveBankRAMRead(m_nSecondAddress, m_nSecondVarSize));
 
         if (m_bInvertBit && m_nSecondVarSize >= ComparisonVariableSize::Bit_0 && m_nSecondVarSize <= ComparisonVariableSize::Bit_7)
             nSecondVal ^= 1;
 
-        return nRetVal * nSecondVal;
+        return static_cast<double>(nRetVal * nSecondVal);
```

Comment:
Warning about the result of a `unsigned int` being cast to `double`, made them `unsigned long long `instead and then cast to `double`.